### PR TITLE
SBOM file generation.

### LINKED ISSUE #279: kwctl: create SBOM file
Each release should have a SBOM file that allows to track the origin of all the dependencies.

### PR DESCRIPTION
**Pull request by jvanz**
## Description

Adds Makefile target and update the release workflow to generate the SBOM file for the kwctl binaries.

Fix #279

### CODE REVIEW

**Review by viccuad — APPROVED** [review]
LGTM!

(there's a bit of repetition between all build jobs, but that was before, and templating and using variables for all of that sounds supercumbersome)

**Review by raulcabello — APPROVED** [review]
LGTM

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -97,6 +96,30 @@ jobs:
           name: kwctl-linux-${{ matrix.targetarch }}
           path: kwctl-linux-${{ matrix.targetarch }}.zip
 
+      - name: Generate SBOM
+        run: |
+          make download-spdx-sbom-generator sbom
```
Comment (flavio) [inline .github/workflows/release.yml:101]: It's a matter of personal taste, but what about using https://github.com/kubewarden/github-actions/tree/v1/sbom-generator-installer to get the sbom tool? Just to be consistent with what we are doing with our policies

**Review by flavio — COMMENTED** [review]
I know this got merged, I'm fine with that. I just have an extra question

File: .github/workflows/release.yml (hunk 1 above)
Comment (jvanz) [reply .github/workflows/release.yml:101]: True. Let me fix it.

**Pull request merged**